src/types.ts

```diff
@@ -608,7 +608,7 @@ export interface OBSEventTypes {
 		/**
 		 * Action performed on the input. See `ObsMediaInputAction` enum
 		 */
-		mediaAction: string;
```

Comment:
As mentioned at the start of this file, this file is auto generated and when there's a new release changes like this will not survive.

Either implement this to obs-websocket repo so [docs/generated/protocol.json](https://github.com/obsproject/obs-websocket/blob/master/docs/generated/protocol.json) has this info that the generator can then read (preferred), or update [scripts/build-types.ts](https://github.com/obs-websocket-community-projects/obs-websocket-js/blob/master/scripts/build-types.ts) so the changes persist